### ISSUE TITLE
Tag count on CucumberTestSuite.java

### ISSUE THREAD

**Issue by edricklacbayo** [serenity-bdd/serenity-cucumber-starter#15]
Is there a way count the number of tags I declared on my CucumberTestSuite?

![image](https://user-images.githubusercontent.com/28752767/77534218-37b53a00-6ed3-11ea-81eb-0eac49b549f1.png)

**Comment by wakaleo** [comment]
Use grep? What is your use case?

**Comment by edricklacbayo** [comment]
At @Before tag I have a counter which increment evertime I run a feature, now let's say I have 1 feature to run, after running the feature at @After I have a method which checks if the counter is equal  to number of tags declared in the CucumberTestSuite, if it is equal then the report will proceed.

**Comment by wakaleo** [comment]
What problem are you trying to solve?

**Comment by edricklacbayo** [comment]
Hi @wakaleo I was trying to get all tags declared in every feature inside the project.

**Comment by wakaleo** [comment]
And what problem will that solve?

**Comment by edricklacbayo** [comment]
I am trying to create a method to list all the tags I declared in the project and then I will check if the tag I am using on my command line is present in the list. If it is. It will trigger an increment in the counter I made so I can use it for reports. I tried using 
![image](https://user-images.githubusercontent.com/28752767/78239350-2e018700-7510-11ea-8775-e73f62d93eec.png)
but it only returns the tags with the scenario I just runned
